lib/aws-cache.rb

```diff
@@ -198,9 +156,22 @@ def has_key?(hash_or_struct, key)
     return false
   end
 
+  def cache_get_2(key, ttl)
```

Comment:
is this functionally distinct from cache_get(), or just rewritten to be shorter/clearer?